src/sum_terms.py

```diff
@@ -0,0 +1,14 @@
+"""Implementation of Sum of nth Term Kata on Code Wars."""
+from __future__ import division
+
+
+def series_sum(n):
+    """Return nth value in series."""
+    if n <= 1:
+        return "%.2f" % n
+    else:
+        count = 1
+        for num in range(n - 1):
+            n = n - 1
```

Comment:
This is an interesting way to solve it, going backwards.  I have not seen one like this.  It might interest you to know that you can actually accomplish this task by using range to count backwards.  The syntax is like range(big number, small number, -1) where -1 indicates a backwards step